pydash/numerical.py

```diff
@@ -383,6 +385,52 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def call_math_operator(value1, value2, op, default):
+    """Return the result of the math operation on the given values."""
+    # TODO: Add defaults for math and div operations.
+    if op in (operator.add, operator.sub):
```

Comment:
This isn't necessary to check. If we want `default` to be optional, then we'd give it a value of `None` and only use the default is it was `not None`.